### DISCUSSION TITLE
Password policy support in GRAFANA

### DISCUSSION THREAD

**Discussion by nadia-aya** [monarc-project/MonarcAppFO#349] [Q&A] [answered]
Hello 

Is there a plan to have a release allowing the  support of a password policy in Grafana? Otherwise how to do sothat we can enforce the security using it

thank you for your answer and help

**Comment by cedricbonhomme** [comment, marked as answer]
Hello,

This is actually not planed. Did you see this [discussion](https://github.com/monarc-project/MonarcAppFO/discussions/347) ?

**Reply by nadia-aya** [reply]
Hello yes I saw it , thank you for your answer

**Reply by cedricbonhomme** [reply]
just by curiosity, what do you want to do with Grafana ? Visualize the password policy or something like this ?

**Reply by ruslanbaidan** [reply]
Could you explain what did you exactly mean in this statement:
"to have a release allowing the support of a password policy in Grafana"?

**Reply by nadia-aya** [reply]
Actually the purpose is to check password policy support feature in new Grafana version, I saw that this topic was already raised in https://github.com/grafana/grafana/issues/12257 and wanted to check if there is an update reagrding that since then..

If the tool doesn't allow the configuration of a password policy that means that we have to do it otherways

**Reply by ruslanbaidan** [reply]
MONARC is not linked with Grafana anyhow.
Are you sure that you addressed the question in the right place?

**Reply by nadia-aya** [reply]
:-) you are right it's my first question here and I missed the suitable channel I will delete it, sorry for the inconvenience

**Reply by ruslanbaidan** [reply]
No problem, thanks for your interest in MONARC project.
